student.rb

```diff
@@ -9,4 +9,9 @@ def initialize(classroom, age, name: 'Unknown', parent_permission: true)
   def play_hooky
     "¯\(ツ)/¯"
   end
+
+  def classroom=(classroom)
+    @classroom = classroom
```

Comment:
- [ ] At the top of your Student class I would recommend you include the getter method for this `@classroom` so it can be fetched, like so:

  ```
  attr_reader :classroom
  ```

Reply:
@TedLivist Thank you for the feedback. I'm sorry you are my first reviewer.
Please for the changes requires which line precisely? Thank you

Reply:
I mean at the top of this class, after `class Classroom`, before `def initialize`, where you define your `attr_reader` in other classes:

```
class Classroom
  attr_accessor :label
  attr_reader :students, :classroom

  def initialize(label)
```